v1/api/swagger/components/schemas/OSServiceImageDownloadHealthSummary.yaml

```diff
@@ -0,0 +1,9 @@
+type: string
+enum:
+  - OK
+  - Warning
+  - Critical
+  - Unknown
+example: OK
+description: >
+  Indicates the download health summary of the OS service image.
```

Comment:
This schema is also used for the overall summary and not only the summary for the individual image. Does this description still apply?

Reply:
As discussed will leave description as is